compute_modules/client/internal_query_client.py

```diff
@@ -100,49 +100,30 @@ def _iterable_to_json_generator(self, iterable: Iterable[Any]) -> Iterable[bytes
         for i in iterable:
             yield json.dumps(i).encode("utf-8")
 
-    @contextmanager
-    def request(
-        self,
-        method: str,
-        url: str,
-        headers: Dict[str, Any],
-        body: Optional[Any] = None,
-    ) -> Generator[http.client.HTTPResponse, Any, None]:
-        """Wrapper for using https connection for requests"""
-        response: Optional[http.client.HTTPResponse] = None
-        connection = http.client.HTTPSConnection(
-            host=self.host,
-            port=self.port,
-            context=self.context,
-            timeout=(60 * 5),  # 5 minutes
-        )
-        try:
-            connection.request(
-                method=method,
-                url=url,
-                body=body,
-                headers=headers,
-            )
-            response = connection.getresponse()
-            yield response
-        finally:
-            connection.close()
+    def init_session(self) -> None:
+        """Initialize requests.Session"""
+        self.session = requests.Session()
```

Comment:
I'm a bit concerned about never explicitly closing the session here, but thinking through it I don't think it's too big of a deal. The only time we would close a session would be when the compute module is stopped, at which point any connections that may have been left open will be closed on the runtime since the runtime will be stopped as well.